### PR TITLE
en: add doc for advanced statefulset

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Add doc for advanced statefulset.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/52
- Other reference link(s):

### CODE REVIEW

File: en/advanced-statefulset.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+category: reference
+---
+
+# Advanced StatefulSet Controller
+
+**Feature Stage**: Alpha
+
+Kubernetes has a built-in [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) that allocates consecutive serial numbers to Pods. For example, when there are three replicas, the Pods are named as pod-0, pod-1, and pod-2. When scaling out or scaling in, you must append a Pod at the end or delete the last pod. For example, when you scale out to four replicas, pod-3 is added. When you scale in to two replicas, pod-2 is deleted.
+
+When you use local storage, Pods are associated with the Nodes storage resources and cannot be scheduled freely. If you want to delete one of the Pods in the middle to maintain its Node but no other Nodes can be migrated, or if you want to delete a Pod that fails and create another Pod with a different serial number, you cannot implement the desired function by the built-in StatefulSet.
```
Comment (TomShawn) [inline en/advanced-statefulset.md]:
```suggestion
When you use local storage, Pods are associated with the Nodes storage resources and cannot be scheduled freely. If you want to delete one of the Pods in the middle to maintain its Node but no other Nodes can be migrated, or if you want to delete a Pod that fails and to create another Pod with a different serial number, you cannot implement such desired function by the built-in StatefulSet.
```

File: en/advanced-statefulset.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+>
+> If the `AdvancedStatefulSet` feature is enabled, TiDB Operator converts the current `StatefulSet` object into an `AdvancedStatefulSet` object. However, after the `AdvancedStatefulSet` feature is disabled, the `AdvancedStatefulSet` object cannot be automatically converted to the built-in `StatefulSet` object of Kubernetes.
+
+## Usage
+
+This section describes how to use the advanced StatefulSet controller.
+
+### View the `AdvancedStatefulSet` Object by kubectl
+
+The data format of `AdvancedStatefulSet` is the same as that of `StatefulSet`, but `AdvancedStatefulSet` is implemented in CRD, with the alias as `asts`. You can view the `AdvancedStatefulSet` object in the namespace by running the following command:
```
Comment (TomShawn) [inline en/advanced-statefulset.md]:
```suggestion
The data format of `AdvancedStatefulSet` is the same as that of `StatefulSet`, but `AdvancedStatefulSet` is implemented based on CRD, with `asts` as the alias. You can view the `AdvancedStatefulSet` object in the namespace by running the following command:
```

**Comment by yikeke** [comment]
Add the cherry-pick label? @ran-huang

**Comment by ran-huang** [comment]
> Add the cherry-pick label? @ran-huang

Sorry I forgot😅

**Pull request merged**